attacked/server.py

```diff
@@ -0,0 +1,56 @@
+import socket
+import non_update_functions
+import logging
+import functions
+from colorama import Fore
+from common.const import *
+from common import protocol
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def run_server() -> socket.socket:

```

Comment:
This dosent run the server.  
It in the best case creates and initialized it.  
consider `create_server`